src/Bicep.Core.Samples/Files/AKS_LF/main.sourcemap.json

```diff
@@ -0,0 +1,286 @@
+    "Item2": 14
+  },
+  "35": {
+    "Item1": "main.bicep",
+    "Item2": 18
+  },
+  "36": {
+    "Item1": "main.bicep",
+    "Item2": 18
+  },
+  "37": {
+    "Item1": "main.bicep",
+    "Item2": 18
+  },
+  "38": {
+    "Item1": "main.bicep",
+    "Item2": 18
+  },
+  "39": {
+    "Item1": "main.bicep",
+    "Item2": 18
+  },
+  "40": {
+    "Item1": "main.bicep",
+    "Item2": 18
+  },
+  "41": {
+    "Item1": "main.bicep",
+    "Item2": 22
+  },
+  "42": {
+    "Item1": "main.bicep",
+    "Item2": 22
+  },
+  "43": {
+    "Item1": "main.bicep",
+    "Item2": 22
+  },
+  "44": {
+    "Item1": "main.bicep",
+    "Item2": 22
+  },
+  "45": {
+    "Item1": "main.bicep",
+    "Item2": 22
+  },
+  "46": {
+    "Item1": "main.bicep",
+    "Item2": 22
+  },
+  "47": {
+    "Item1": "main.bicep",
+    "Item2": 24
+  },
+  "48": {
+    "Item1": "main.bicep",
+    "Item2": 24
+  },
+  "49": {
+    "Item1": "main.bicep",
+    "Item2": 24
+  },
+  "50": {
+    "Item1": "main.bicep",
+    "Item2": 24
+  },
+  "53": {
+    "Item1": "main.bicep",
+    "Item2": 27
+  },
+  "54": {
+    "Item1": "main.bicep",
+    "Item2": 27
+  },
+  "55": {
+    "Item1": "main.bicep",
+    "Item2": 27
+  },
+  "56": {
+    "Item1": "main.bicep",
+    "Item2": 27
+  },
+  "57": {
+    "Item1": "main.bicep",
+    "Item2": 29
+  },
+  "58": {
+    "Item1": "main.bicep",
+    "Item2": 30
+  },
+  "59": {
+    "Item1": "main.bicep",
+    "Item2": 31
+  },
+  "60": {
+    "Item1": "main.bicep",
+    "Item2": 32
+  },
+  "61": {
+    "Item1": "main.bicep",
+    "Item2": 33
+  },
+  "62": {
+    "Item1": "main.bicep",
+    "Item2": 34
+  },
+  "63": {
+    "Item1": "main.bicep",
+    "Item2": 35
+  },
+  "64": {
+    "Item1": "main.bicep",
+    "Item2": 36
+  },
+  "65": {
+    "Item1": "main.bicep",
+    "Item2": 37
+  },
+  "66": {
+    "Item1": "main.bicep",
+    "Item2": 38
+  },
+  "67": {
+    "Item1": "main.bicep",
+    "Item2": 33
+  },
+  "68": {
+    "Item1": "main.bicep",
+    "Item2": 32
+  },
+  "69": {
+    "Item1": "main.bicep",
+    "Item2": 41
+  },
+  "70": {
+    "Item1": "main.bicep",
+    "Item2": 42
+  },
+  "71": {
+    "Item1": "main.bicep",
+    "Item2": 43
+  },
+  "72": {
+    "Item1": "main.bicep",
+    "Item2": 44
+  },
+  "73": {
+    "Item1": "main.bicep",
+    "Item2": 45
+  },
+  "74": {
+    "Item1": "main.bicep",
+    "Item2": 46
+  },
+  "75": {
+    "Item1": "main.bicep",
+    "Item2": 45
+  },
+  "76": {
+    "Item1": "main.bicep",
+    "Item2": 44
+  },
+  "77": {
+    "Item1": "main.bicep",
+    "Item2": 43
+  },
+  "78": {
+    "Item1": "main.bicep",
+    "Item2": 41
+  },
+  "79": {
+    "Item1": "main.bicep",
+    "Item2": 51
+  },
+  "80": {
+    "Item1": "main.bicep",
+    "Item2": 52
+  },
+  "81": {
+    "Item1": "main.bicep",
+    "Item2": 53
+  },
+  "82": {
+    "Item1": "main.bicep",
+    "Item2": 51
+  },
+  "83": {
+    "Item1": "main.bicep",
+    "Item2": 30
+  },
+  "84": {
+    "Item1": "main.bicep",
+    "Item2": 27
+  }
```

Comment:
This format is concise, but I do think it would be worth thinking about making it more user-readable. We've had similar feedback for people trying to consume our custom [types JSON format](https://github.com/Azure/bicep-types-az/tree/main/generated).

What about something like:
```json
{ "path": "main.bicep", "sourceLine": 84, "targetLine": 27 }
```

There's also a lot of repetition with the file path (and these can definitely get pretty long) - what about instead grouping by filename:
```json
[
{
  "filePath": "main.bicep",
  "sourceMap": [
    { "sourceLine": 123, "targetLine": 456 }
  ]
},
{
  "filePath": "myModule.bicep",
  "sourceMap": [
    { "sourceLine": 12, "targetLine": 34 }
  ]
},
]
```

Reply:
Finally, how are relative file paths handled? If we have a module reference to `../other.bicep`, which has its own reference to `../andAnother.bicep`, will we output `../../andAnother.bicep` in this file?

Reply:
@anthony-c-martin it is for most people it is easier & quicker to read on separate lines than all in 1 line as it reduces the amount of eye travelling across the screen to get the same data which means its physically more beneficial and will reduce eye strain which in the longer term is a huge health benefit that reduces the likelihood of repetitive strain injuries which we should do as much as we can to ensure that the language doesn't unnesseccarily cause issues that could easily be avoided. We should also not deny the option of writing on a single line as not to stop those that prefer writing in that way, but suggest as a best practice to split on to multiple lines for accessibility reasons,

Reply:
No concerns about emitting an indented file, but we definitely need the properties to have more descriptive names than `Item1` or `Item2`.